src/Datadog.Trace.ClrProfiler.Managed/Emit/MethodBuilder.cs

```diff
@@ -31,6 +32,7 @@ internal class MethodBuilder<TDelegate>
         private string _concreteTypeName;
         private object[] _parameters = new object[0];
         private Type[] _explicitParameterTypes = null;
+        private string[] _namespaceNameFilters = null;
```

Comment:
Can we call this `_namespaceFilters`? "Namespace name" seems redundant.

Reply:
I figured Namespace + Name should be apparent.
Full name was ambiguous, because that can include assembly and version.

Reply:
My mistake. I thought `namespaceName` meant "the name of the namespace," which is just weird.